source/java/product-guide/index.markdown

```diff
@@ -1757,14 +1757,14 @@ For example, assume that an account store represented by mapping `$accountStoreM
 
 Applications cannot store Accounts directly - Accounts are always stored in an Account Store (a Directory or Group). Therefore, if you would like an application to be able to create new accounts, you must specify which of the application's associated account stores should store the application's newly created accounts.  This designated account store is called the application's _default account store_.
 
-You specify an application's default account store by setting the AccountStoreMapping's `isDefaultAccountStore` attribute to equal `true`.  You can do this when you create the `accountStoreMapping` resource.  Or if the resource has already been created:
+You specify an application's default account store by setting the ApplicationAccountStoreMapping's `isDefaultAccountStore` attribute to equal `true`.  You can do this when you create the `applicationAccountStoreMapping` resource.  Or if the resource has already been created:
 
-1. Find the `accountStoreMapping` resource in the Application's `accountStoreMappings` [collection](#collections) that reflects the `accountStore` you wish to be the application's default account store.
-2. Issue an update request to that AccountStoreMapping's instance with `defaultAccountStore` set to `true`.
+1. Find the `applicationAccountStoreMapping` resource in the Application's `accountStoreMappings` [collection](#collections) that reflects the `accountStore` you wish to be the application's default account store.
+2. Issue an update request to that ApplicationAccountStoreMapping's instance with `defaultAccountStore` set to `true`.
 
 **Example Request**
 
-    accountStoreMapping.setDefaultAccountStore(Boolean.TRUE).save();
+    applicationAccountStoreMapping.setDefaultAccountStore(Boolean.TRUE).save();
 
 Now, any time a new account is created from an application's `createAccount` method, the account will actually be created in the designated default account store.
 
```

Comment:
In the "List Account Store Mappings" section it should say:

The response is a paginated list of `ApplicationAccountStoreMapping` resources.  You may use collection [pagination](#pagination) query parameters to customize the paginated response.
 
 **Example Request**
 
     ApplicationAccountStoreMappingList accountStoreMappings = application.getAccountStoreMappings();
     for (ApplicationAccountStoreMapping accountStoreMapping : accountStoreMappings) {
     	System.out.println(accountStoreMapping.getAccountStore().getHref());
     }